apps/web/src/pages/activities/components/ActivityGraphGlobalStyles.tsx

```diff
@@ -0,0 +1,66 @@
+import { css, Global } from '@emotion/react';
+import { colors } from '../../../design-system';
+
+export function ActivityGraphGlobalStyles({ isTriggerSent, isDark }: { isTriggerSent: boolean; isDark: boolean }) {
+  return <Global styles={ChartStyles(isTriggerSent, isDark)} />;
+}
+
+function ChartStyles(isTriggerSent: boolean, isDark: boolean) {
```

Comment:
```suggestion
function chartStyles(isTriggerSent: boolean, isDark: boolean) {
```
Would suggest doing this camelCasing, so we can keep pascal casing for classes in Components